### PR TITLE
Variable translation broke the number of "following" in pt-BR

### PR DESCRIPTION
**Pull request by maisondasilva**
Variable translation broke the number of "following" in pt-BR
I believe need to be more careful when translating to avoid variables being translated, breaking the website experience!

Imagem

### CODE REVIEW

**Comment by maisondasilva** [comment]
@pfrazee @ansh

File: src/locale/locales/pt-BR/messages.po [hunk 1]
```diff
@@ -1,6 +1,6 @@
 msgid ""
 msgstr ""
-"POT-Creation-Date: 2023-12-22 01:56+0530\n"
+"POT-Creation-Date: 2024-02-07 00:25+0530\n"
```
Comment (surfdude29) [inline src/locale/locales/pt-BR/messages.po:3]: Sorry to jump in, I think you meant to update `PO-Revision-Date` instead of the creation date?

Comment (maisondasilva) [reply src/locale/locales/pt-BR/messages.po]: Thanks for the warning, for some reason the editor I use changed the wrong line, I already corrected it.

Comment (surfdude29) [reply src/locale/locales/pt-BR/messages.po]: No worries :)

**Review by pfrazee — APPROVED** [review]
🙏

**Pull request merged**